pypeclub/OpenPype
# TrayPublisher: copied ValidateFrameRange from SP

## Brief description
Validate frame range was missing from TrayPublisher

## Additional info
SP will be removed in the future, copied this to TrayPublisher to keep functionality.


## Testing notes:
1. publish render sequence with non matching frame count against the DB
2. Validator should fail
3. check that validator could be disabled
